dynamic-plugins/wrappers/backstage-community-plugin-tekton/package.json

```
@@ -29,7 +28,7 @@
     "export-dynamic:clean": "janus-cli package export-dynamic-plugin --in-place --clean"
   },
   "dependencies": {
-    "@janus-idp/backstage-plugin-tekton": "3.15.0"
+    "@backstage-community/plugin-tekton": "3.16.0"
```

Workaround for style-inject issue is released in new patch version.

```suggestion
    "@backstage-community/plugin-tekton": "3.16.1"
```